scripts/quests/windurst/Scooped.lua

```diff
@@ -0,0 +1,82 @@
+-----------------------------------
+-- Scooped!
+-----------------------------------
+-- !addquest 2 38
+-- Hariga-Origa : !pos -70.244, -3.800, -4.439
+-----------------------------------
+
+local quest = Quest:new(xi.questLog.WINDURST, xi.quest.id.windurst.SCOOPED)
+
+quest.reward =
+{
+    gil = 1500,
+}
+
+quest.sections =
+{
+    {
+        check = function(player, status, vars)
+            return status == xi.questStatus.QUEST_AVAILABLE and
+                player:hasCompletedQuest(xi.questLog.WINDURST, xi.quest.id.windurst.MAKING_HEADLINES)
+        end,
+
+        [xi.zone.WINDURST_WATERS] =
+        {
+            ['Naiko-Paneiko'] = quest:progressEvent(676),
+
+            onEventFinish =
+            {
+                [676] = function(player, csid, option, npc)
+                    quest:begin(player)
+                end,
+            },
+        },
+    },
+
+    {
+        check = function(player, status, vars)
+            return status == xi.questStatus.QUEST_ACCEPTED
+        end,
+
+        [xi.zone.WINDURST_WATERS] =
+        {
+            ['Naiko-Paneiko'] =
+            {
+                onTrade = function(player, npc, trade)
+                    if npcUtil.tradeHasExactly(trade, xi.item.BRONZE_BOX) then
+                        return quest:progressEvent(680)
+                    end
+                end,
+
+                onTrigger = function(player, npc)
+                    return quest:progressEvent(677)
```

Comment:
:event(677)

Reply:
Also, this could be simplified further to just `onTrigger = quest:event(677),`